docs/cli.md

```diff
@@ -152,710 +163,464 @@ keytabFile=file.keytab
 
-```bash
-gcli user delete --user new_user
-```
+`gcli user list`
+: List all users
 
 ### Group commands
 
-#### Create a group
+`gcli group create --group <group>`
+: Create a group.
 
-```bash
-gcli group create --group new_group
-```
+`gcli group details --group <group>`
+: Display the details about a group.
 
-#### Display a group's details
+`gcli group list`
+: List all groups.
 
-```bash
-gcli group details --group new_group
-```
-
-#### List all groups
-
-```bash
-gcli group list
-```
-
-#### Delete a group
-
-```bash
-gcli group delete --group new_group
-```
+`gcli group delete --group <group>`
+: Delete a group.
 
 ### Tag commands
 
-#### Display a tag's details
+`gcli tag create --tag <tag-A> <tagB>`
+: Create tags.
 
-```bash
-gcli tag details --tag tagA
-```
-
-#### Create tags
+`gcli tag details --tag <tag>`
+: Display the details about a tag.
 
-```bash
- gcli tag create --tag tagA tagB
- ```
+`gcli tag list`
+: List all tags.
 
-#### List all tag
-
-```bash
-gcli tag list
-```
-
-#### Delete tags
-
-```bash
-gcli tag delete --tag tagA tagB
-```
+`gcli tag delete --tag <tag-A> <tag-B>`
+: Delete tags.
 
-#### Add tags to an entity
+`gcli tag set --name <entity> --tag tagA tagB`
+: Add tags to an entity. The type of the entity is implied by its name.
 
-```bash
-gcli tag set --name catalog_postgres.hr --tag tagA tagB
-```
+`gcli tag remove --name <entity> --tag <tagA> <tagB>`
+: Remove tags from an entity. The type of the entity is implied by its name.
 
-#### Remove tags from an entity
+`gcli tag remove --name <entity>`
+: Remove all tags from an entity. The type of the entity is implied by its name.
 
-```bash
-gcli tag remove --name catalog_postgres.hr --tag tagA tagB
-```
+`gcli tag list --name <entity>`
+: List all tags on an entity
 
-#### Remove all tags from an entity
+`gcli tag properties --tag <tag>`
+: List the properties of a tag.
 
-```bash
-gcli tag remove --name catalog_postgres.hr
-```
+`gcli tag set --tag <tag> --property <property> --value <value>`
+: Set a property for a tag.
 
-#### List all tags on an entity
+`gcli tag remove --tag <tag> --property <property>`
+: Delete a property from a tag.
 
-```bash
-gcli tag list --name catalog_postgres.hr
-```
+`gcli tag update --tag <tag> --rename <new-name>`
+: Rename a tag.
 
-#### List the properties of a tag
-
-```bash
-gcli tag properties --tag tagA
-```
-
-#### Set a properties of a tag
-
-```bash
-gcli tag set --tag tagA --property test --value value
-```
-
-#### Delete a property of a tag
-
-```bash
-gcli tag remove --tag tagA --property test
-```
-
-#### Rename a tag
-
-```bash
-gcli tag update --tag tagA --rename newTag
-```
-
-#### Update a tag's comment
-
-```bash
-gcli tag update --tag tagA --comment "new comment"
-```
+`gcli tag update --tag <tag> --comment "<new-comment>"`
+: Update the comment for a tag.
 
 ### Owner commands
 
-#### List an owner
+`gcli catalog details --name <catalog> --owner`
+: Show the details about an owner for a catalog.
```

Comment:
"an owner for a catalog." - > "the owner of a catalog."

Reply:
Correct. Got it.